sources/Context.cpp

```diff
@@ -105,6 +110,27 @@ Context::process() {
   return true;
 }
 
+void
+Context::filterDynModels() {
```

Comment:
that would be filterDynamicAutomaton

Reply:
No we filter the dynamic models

Reply:
in the method we go throught the dynamic automatons though

Reply:
It is a mislead of the loop. In reality we iterate on dynamicModels_.models. It is changed

Reply:
Yes but we erase dynamic models from the model list if they are not completly connected. the dynamic models are filtered

Reply:
filterPartiallyConnectedDynamicModels?

Reply:
OK